### PR TITLE
[Feature] Show a warning message when the user is about to join the guest room conversation

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

To show a warning message when the user is about to join the conversation with the current account using the guestroom link.

Needs releases with:

- [ wireapp/wire-ios-sync-engine/pull/1453 ]

### Notes

https://wearezeta.atlassian.net/browse/SQSERVICES-506

### CODE REVIEW

File: Wire-iOS/Sources/URLActionRouter.swift [hunk 1]
```diff
@@ -173,19 +176,19 @@ extension URLActionRouter: PresentationDelegate {
         NotificationCenter.default.post(name: .companyLoginDidFinish, object: self)
     }
 
-    private func presentConnectBotAlert(with decisionHandler: @escaping (Bool) -> Void) {
-        let alert = UIAlertController(title: "url_action.title".localized,
-                                      message: "url_action.connect_to_bot.message".localized,
+    private func presentConfirmAlert(title: String, message:String, decisionHandler: @escaping (Bool) -> Void) {
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/URLActionRouter.swift:179]: Suggestion: `presentConfirmationAlert`

File: Wire-iOS/Sources/URLActionRouter.swift [hunk 2]
```diff
@@ -173,19 +176,19 @@ extension URLActionRouter: PresentationDelegate {
...
+        let alert = UIAlertController(title: title,
+                                      message: message,
                                       preferredStyle: .alert)
 
-        let agreeAction = UIAlertAction(title: "url_action.confirm".localized,
+        let agreeAction = UIAlertAction(title: L10n.Localizable.UrlAction.confirm,
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/URLActionRouter.swift:184]: You can also use the existing helper: `UIAlertAction.confirm(style: .default) { _ in decisionHandler(true) } `

File: Wire-iOS/Sources/URLActionRouter.swift [hunk 3]
```diff
@@ -173,19 +176,19 @@ extension URLActionRouter: PresentationDelegate {
...
                                         style: .default) { _ in
                                             decisionHandler(true)
         }
 
         alert.addAction(agreeAction)
 
-        let cancelAction = UIAlertAction(title: "general.cancel".localized,
+        let cancelAction = UIAlertAction(title: L10n.Localizable.General.cancel,
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/URLActionRouter.swift:191]: Also here there is an existing helper: `UIAlertAction.cancel({ decisionHandler(false) })`

File: Wire-iOS/Sources/URLActionRouter.swift (hunk 1 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/URLActionRouter.swift]: Fixed

File: Wire-iOS/Sources/URLActionRouter.swift (hunk 2 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/URLActionRouter.swift]: Done

File: Wire-iOS/Sources/URLActionRouter.swift (hunk 3 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/URLActionRouter.swift]: Done

**Review by johnxnguyen — APPROVED** [review]

**Review by David-Henner — APPROVED** [review]

**Pull request merged**